metakb/main.py

```diff
@@ -0,0 +1,61 @@
+"""Main application for FastAPI."""
+from fastapi import FastAPI, Query
+from fastapi.openapi.utils import get_openapi
+from metakb.query import QueryHandler
+from metakb.schemas import SearchService
+from typing import Optional
+
+app = FastAPI(docs_url='/api/v2', openapi_url='/api/v2/openapi.json')
+query = QueryHandler(uri="bolt://localhost:7687",
+                     credentials=("neo4j", "admin"))
```

Comment:
We might want to check the environment for these values instead of forcing them to be hard-coded (could use METAKB_DB_URL, METAKB_DB_USERNAME, METAKB_DB_PASSWORD eg)

Reply:
Forgot we added this in the CLI! Added